### PR TITLE
Add missing System.Net AppContext switches documentation

### PR DESCRIPTION
**Pull request by davidsh**
Ref: internal bug 462936

### CODE REVIEW

File: docs/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element.md [hunk 1]
```diff
@@ -76,7 +76,11 @@ Defines one or more switches used by the <xref:System.AppContext> class to provi
...
-|`Switch.System.Net.`<br/>`DontEnableSchUseStrongCrypto`|Controls whether the <xref:System.Net.ServicePointManager?displayProperty=nameWithType> and <xref:System.Net.Security.SslStream?displayProperty=nameWithType> classes can use the SSL 3.0 protocol. For more information, see [Mitigation: TLS Protocols](~/docs/framework/migration-guide/mitigation-tls-protocols.md).|.NET Framework 4.6|  
+|`Switch.System.Net.`<br/>`DontCheckCertificateEKUs`|Disables certificate EKU OID validation.|.NET Framework 4.6|
+|`Switch.System.Net.`<br/>`DontEnableSchSendAuxRecord`|Disables TLS1.0 BEAST mitigation by disabling the use of SCH_SEND_AUX_RECORD.|.NET Framework 4.6|
+|`Switch.System.Net.`<br/>`DontEnableSchUseStrongCrypto`|Controls whether the <xref:System.Net.ServicePointManager?displayProperty=nameWithType> and <xref:System.Net.Security.SslStream?displayProperty=nameWithType> classes can use the SSL 3.0 protocol. For more information, see [Mitigation: TLS Protocols](~/docs/framework/migration-guide/mitigation-tls-protocols.md).|.NET Framework 4.6|
```
Comment (stevewhims) [inline docs/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element.md]: Based on your other comment, should that be "TLS/SSL 3.0 protocol"?

**Review by stevewhims — APPROVED** [review]
Looks good. I just left one question about whether "SSL" should be "TLS/SSL".

File: docs/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element.md (hunk 1 above)
Comment (davidsh) [reply docs/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element.md]: >Based on your other comment, should that be "TLS/SSL 3.0 protocol"?

No.  SSL 3.0 is a protocol.  TLS 1.0, TLS 1.1, TLS 1.2 are protocols.  TLS 1.0 is the successor to SSL 3.0.

**Comment by rpetrusha** [comment]
PR does not appear in the OPS Portal. Closing and reopening to begin new build.

**Comment by mairaw** [comment]
Still not showing. Kicking off a new build.

File: docs/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element.md [hunk 2]
```diff
@@ -76,7 +76,11 @@ Defines one or more switches used by the <xref:System.AppContext> class to provi
...
 |`Switch.System.`<br/>`MemberDescriptorEqualsReturnsFalseIfEquivalent`|Controls whether a test for equality compares the <xref:System.ComponentModel.MemberDescriptor.Category%2A?displayProperty=nameWithType> property of one object with the <xref:System.ComponentModel.MemberDescriptor.Description%2A?displayProperty=nameWithType> property of the second object. For more information, see [Mitigation: MemberDescriptor.Equals](~/docs/framework/migration-guide/mitigation-memberdescriptor-equals.md).|.NET Framework 4.6.2|  
-|`Switch.System.Net.`<br/>`DontEnableSchUseStrongCrypto`|Controls whether the <xref:System.Net.ServicePointManager?displayProperty=nameWithType> and <xref:System.Net.Security.SslStream?displayProperty=nameWithType> classes can use the SSL 3.0 protocol. For more information, see [Mitigation: TLS Protocols](~/docs/framework/migration-guide/mitigation-tls-protocols.md).|.NET Framework 4.6|  
+|`Switch.System.Net.`<br/>`DontCheckCertificateEKUs`|Disables certificate EKU OID validation.|.NET Framework 4.6|
```
Comment (mairaw) [inline docs/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element.md]: @davidsh we usually have to define what acronyms are on the first use. Should we change this from EKU OID to extended key usage (EKU) object identifier (OID) or does it get too noisy?

File: docs/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element.md [hunk 3]
```diff
@@ -76,7 +76,11 @@ Defines one or more switches used by the <xref:System.AppContext> class to provi
...
+|`Switch.System.Net.`<br/>`DontEnableSchSendAuxRecord`|Disables TLS1.0 BEAST mitigation by disabling the use of SCH_SEND_AUX_RECORD.|.NET Framework 4.6|
```
Comment (mairaw) [inline docs/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element.md]: same here. do we need to explain a bit more what BEAST is?

**Review by mairaw — APPROVED** [review]
LGTM. I had just a few questions before merging.

File: docs/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element.md (hunk 2 above)
Comment (davidsh) [reply docs/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element.md]: >Should we change this from EKU OID to extended key usage (EKU) object identifier (OID)

I was leaving that up to you.  We probably should be standardizing how we talk about these kinds of terminology.  EKU OID are .NET concepts:
https://msdn.microsoft.com/en-us/library/system.security.cryptography.x509certificates.x509enhancedkeyusageextension(v=vs.110).aspx

"An enhanced key usage (EKU) extension is a collection of object identifiers (OIDs) that indicate the ..."

So, feel free to expand how we are describing this.

File: docs/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element.md (hunk 3 above)
Comment (davidsh) [reply docs/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element.md]: This describes BEAST (Browser Exploit Against SSL/TLS):

Browser
Exploit
Against
SSL
TLS

File: docs/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element.md (hunk 2 above)
Comment (mairaw) [reply docs/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element.md]: Got the wrong name, sorry. Ok, the standard process is to expand them on first use, so I'll make that fix here. Thanks @davidsh!

File: docs/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element.md (hunk 3 above)
Comment (davidsh) [reply docs/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element.md]: Feel free to expand the acronym as well as add hyperlink to more explanatory material.

**Comment by davidsh** [comment]
>o I'll make that fix here

Since this is my PR, are you @mairaw able to add commits to it or do I need to revise my PR and add the commits to it based on this feedback.

**Comment by mairaw** [comment]
As a repo maintainer, I usually have permissions to make edits (unless you explicitly choose not to when submitting a PR @davidsh)

**Comment by davidsh** [comment]
@mairaw Are we going to merge this?  And then will you make additional changes based on the discussion above?

**Comment by stevewhims** [comment]
I just merged it. Maira, did you make those changes already? If not, let’s do another PR.

**Comment by stevewhims** [comment]
I’ll create a new PR with those changes, and I’ll add david as reviewer.

From: Steve White
Sent: Wednesday, October 4, 2017 11:18 AM
To: 'dotnet/docs' <reply@reply.github.com>; dotnet/docs <docs@noreply.github.com>
Cc: Review requested <review_requested@noreply.github.com>
Subject: RE: [dotnet/docs] Add missing System.Net AppContext switches documentation (#3270)

I just merged it. Maira, did you make those changes already? If not, let’s do another PR.

**Pull request merged**